pkg/autodiscovery/providers/telemetry.go

```diff
@@ -0,0 +1,49 @@
+package providers
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/DataDog/datadog-agent/pkg/autodiscovery/integration"
+	"github.com/DataDog/datadog-agent/pkg/autodiscovery/providers/names"
+	pkgconfig "github.com/DataDog/datadog-agent/pkg/config"
+)
+
+type TelemetryProvider struct{}
+
+func NewTelemetryProvider() *TelemetryProvider {
+	return &TelemetryProvider{}
+}
+
+func (c *TelemetryProvider) Collect(ctx context.Context) ([]integration.Config, error) {
+	port := pkgconfig.Datadog.GetString("expvar_port")
+
+	instance := fmt.Sprintf(`
+openmetrics_endpoint: http://localhost:%s/telemetry
+namespace: datadog.agent
+no_index: true
+metrics:
+- logs_sender_latency.*
+- logs_sent.*
+- logs_dropped.*
+- payload_drops.*`, port)
+
+	return []integration.Config{
+		{
+			Name:      "openmetrics",
```

Comment:
Can we create a new integration `datadog_agent` or  `datadog_agent_telemetry` in integrations-core based on the `openmetrics` base check? 

The idea is to make it more clear in the agent status, that the check is an official datadog integration, because usually `openmetrics` checks are considered by customer as "custom check" that generate custom metrics.

If you need an example we did one integration for the `cluster-agent`: https://github.com/DataDog/integrations-core/tree/master/datadog_cluster_agent